Add example landing docs and media

dafortune opened the pull request:
This must exist for landing pages to work properly.
To be modified when the real landing pages gets shipped.

dschenkelman commented:
@cristiandouce this is not mergeable. we can do it tomorrow after rebasing

dschenkelman commented on themes/default/views/doc.nosidebar.jade:
```diff
@@ -7,11 +7,9 @@ html(lang="en")
-    link(rel="shortcut icon", href="/img/logo-small.png")
-    link(href="//cdn.auth0.com/styleguide/1.0/index.css", rel="stylesheet")
-    link(href="/docs/css/docs.css", rel="stylesheet")
+    link(rel="shortcut icon", href="//cdn.auth0.com/styleguide/latest/lib/logos/img/favicon.png")
```
@cristiandouce future PR with STYLEGUIDE_VERSION for this and auth0-manage

dafortune committed:
Add example landing docs and media

This must exist for landing pages to work properly.
To be modified when the real landing pages gets shipped.
